components/ShadowLayer/src/MDCShadowLayer.m

```diff
@@ -299,6 +301,17 @@ - (void)setElevation:(CGFloat)elevation {
   _bottomShadow.shadowOpacity = shadowMetrics.bottomShadowOpacity;
 }
 
+- (void)setCornerRadius:(CGFloat)cornerRadius {
+  BOOL radiusChanged = !MDCCGFloatEqual(self.cornerRadius, cornerRadius);
```

Comment:
Question: In a setter shouldn't we use the ivar?

Reply:
Yes we should.

Reply:
Drive-by comment: Let's please add a unit test for this method. To make sure there isn't an infinite loop. :)

Reply:
Yes, typically we prefer to use the ivar in init* methods and getters / setters.
However, in this case, cornerRadius is exposed by CALayer, our super class, so we don't have direct access to the internal data structure storing cornerRadius.

Reply:
Maybe next time it can make things clearer to use `super.cornerRadius`?